src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -631,6 +638,9 @@ private class InternalBackupResult
                 catch (Exception e)
                 {
                     status.LocalBackup.Exception = e.ToString();
+
+                    // deleting the temp backup file if the backup failed
+                    IOExtensions.DeleteFile(tempBackupFilePath);
```

Comment:
Maybe another try/catch here.
We explicitly don't care if we failed to delete the file, no?

Reply:
done